pkg/cmd/root/help_topic.go

```diff
@@ -86,8 +86,12 @@ var HelpTopics = []helpTopic{
 			available in the viewport. When the value is a percentage, it will be applied against
 			the number of columns available in the current viewport.
 
-			%[1]sGH_NO_UPDATE_NOTIFIER%[1]s: set to any value to disable update notifications. By default, gh
-			checks for new releases once every 24 hours and displays an upgrade notice on standard
+			%[1]sGH_NO_UPDATE_NOTIFIER%[1]s: set to any value to disable GitHub CLI update notifications.
+			By default, gh checks for new releases once every 24 hours and displays an upgrade notice on standard
+			error if a newer version was found.
+
+			%[1]sGH_NO_EXTENSION_UPDATE_NOTIFIER%[1]s: set to any value to disable GitHub CLI extension update notifications.
```

Comment:
Also no file changed here for me to comment on but this currently looks like:

```
A new release of sonar is available: 81a4ce86e027f31d306883c25a71b5d05b007e2e → 9977d8e535f98d57b15f0b5cb1dc887205acd2e4
To upgrade, run: gh extension upgrade sonar
https://github.com/andyfeller/gh-sonar.git
```

"release" is maybe not the most accurate term for this.

It does make me think that the current upgrade approach around `GitKind` extensions is probably not very friendly, with a potential notification every 24 hours for any new git commits, and no obvious way to understand whether the git commit is interesting. Non-blocking.